src/vcpkg/commands.extract.cpp

```diff
@@ -0,0 +1,30 @@
+#include <vcpkg/base/fwd/message_sinks.h>
+
+#include <vcpkg/archives.h>
+#include <vcpkg/commands.extract.h>
+#include <vcpkg/commands.h>
+#include <vcpkg/vcpkgcmdarguments.h>
+#include <vcpkg/vcpkgpaths.h>
+
+namespace vcpkg::Commands
+{
+    const CommandStructure ExtractCommandStructure = {
+        [] { return msg::format(msgExtractHelp); },
+        2,
+        2,
+        {{}, {}, {}},
```

Comment:
As discussed offline, this probably needs some argument to "override" the format detection to support unzipping formats like `vsix`, `nupkg`, etc.